src/main/interface/cli.c

```diff
@@ -3075,6 +3075,24 @@ static void cliDefaults(char *cmdline)
     }
 }
 
+void cliPrintVarDefault(const clivalue_t *value)
+{
+    const pgRegistry_t *pg = pgFind(value->pgn);
+    if (pg) {
+        const char *defaultFormat = "Default value: ";
+        const int valueOffset = getValueOffset(value);
+        memcpy(pg->copy, pg->address, pg->size);
```

Comment:
Should we abstract these into `backupConfig(pg)` and `resetConfig(pg)` (and reuse these in `backupConfigs()` / `resetConfigs()`?

Reply:
Backup/restore of an individual PG have been abstracted and used in both places.